apps/masterbots.ai/lib/hooks/useSonner.tsx

```
@@ -0,0 +1,84 @@
+import { SuccessIcon, InfoIcon, ErrorIcon, CloseIcon } from '@/components/icons'
+import { cn } from '@/lib/utils'
+import { toast } from 'sonner'
+
+export type customSonnerProps = {
+  type?: 'success' | 'error' | 'info'
+  text?: string | null
+}
+
+export function useSonner() {
+  let config: {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)